release_notes/Service_Fabric_ReleaseNotes_82CU21.md

```diff
@@ -0,0 +1,79 @@
+# Microsoft Azure Service Fabric 8.2 Second Refresh First Update Release Notes
```

Comment:
I think this would be easier for customers to track this as a third refresh rather than a first update. Even if this release just addresses the regression, I think it would be easier to communicate through a different name.

Reply:
People are waiting for 9.0

Reply:
> People are waiting for 9.0

Coming soon!